Makefile

```diff
@@ -93,7 +93,7 @@ all: \
 
 base:
 	docker build $(DOCKER_BUILD_FLAGS) -t $(DOCKER_PREFIX)$@:$(CAPELLA_DOCKERIMAGES_REVISION) base
-	$(MAKE) PUSH_IMAGES=$(PUSH_IMAGES) IMAGENAME=$@ .push
+	$(MAKE) PUSH_IMAGES=$(PUSH_IMAGES) DOCKER_TAG=$(CAPELLA_DOCKERIMAGES_REVISION) IMAGENAME=$@ .push
```

Comment:
What about the colon `:` in the `.push` recipe that comes before? `$(DOCKER_TAG)`?

Reply:
What do you mean exactly? It is the separation between image name and tag.

Reply:
Ah, mixed sth up I'd say.